tests/python/topi/python/test_topi_conv1d_transpose_ncw.py

```diff
@@ -94,6 +94,9 @@ def test_conv1d_transpose_ncw():
     verify_conv1d_transpose_ncw(1, 1, 10, 1, 5, 1, (0, 3), (0,))
     verify_conv1d_transpose_ncw(1, 1, 10, 1, 5, 1, (1, 3), (0,))
     verify_conv1d_transpose_ncw(1, 1, 10, 1, 5, 1, (2, 3), (0,))
+    verify_conv1d_transpose_ncw(1, 257, 128, 1, 512, 128, 256, (0,))
+    verify_conv1d_transpose_ncw(1, 257, 128, 2, 512, 128, 256, (0,))
+    verify_conv1d_transpose_ncw(1, 257, 128, 257, 512, 128, 256, (0,))
```

Comment:
Could you add a test for the case kernel_size==stride? I think this is common in encoder/decoder networks

Reply:
Added more tests with kernel_size=stride